### PR TITLE
Enable readme link synchronization

### PR DESCRIPTION
**Pull request by qqaatw**
# What does this PR do?

This PR enables readme links synchronization between `README.md` and each localized README.

## Who can review?

@sgugger

### CODE REVIEW

File: utils/check_copies.py [hunk 1]
```diff
@@ -352,12 +354,17 @@ def _rep(match):
...
         if title not in localized_model_index:
             num_models_equal = False
             # Add an anchor white space behind a model description string for regex.
             # If metadata cannot be captured, the English version will be directly copied.
             localized_model_index[title] = re.sub(_re_capture_meta, _rep, model + " ")
+        else:
+            # Synchronize link
+            localized_model_index[title] = re.sub(
+                _re_capture_title_link, f"**[{title}]({model_link})**", localized_model_index[title], count=1
+            )
```
Comment (sgugger) [inline utils/check_copies.py:367]:
```suggestion
            localized_model_index[title] = _re_capture_title_link.sub(
                f"**[{title}]({model_link})**", localized_model_index[title], count=1
            )
```

File: utils/check_copies.py [hunk 2]
```diff
@@ -430,15 +437,17 @@ def check_model_list_copy(overwrite=False, max_per_line=119):
...
+        if filename == "README.md":
+            continue
+        if overwrite:
+            _, start_index, end_index, lines = _find_text_in_file(
+                filename=os.path.join(REPO_PATH, filename), start_prompt=_start_prompt, end_prompt=_end_prompt
+            )
+            with open(os.path.join(REPO_PATH, filename), "w", encoding="utf-8", newline="\n") as f:
+                f.writelines(lines[:start_index] + [converted_md] + lines[end_index:])
+        else:
+            if not num_models_equal:
```
Comment (sgugger) [inline utils/check_copies.py:450]: Maybe join this in an `elif`?

**Review by sgugger — APPROVED** [review]
Thanks for fixing this util!

**Review by JetRunner — APPROVED** [review]
LGTM! Thanks @qqaatw for taking care of this.

**Review by LysandreJik — APPROVED** [review]
That's great work @qqaatw. thank you!

**Pull request merged**